js/miniautures.js

```diff
@@ -0,0 +1,28 @@
+import  {urlForExport, likesForExport, commentsForExport} from './data.js';
+
+const pictures = document.querySelector ('.pictures');
+const pictureTemplate = document.querySelector ('#picture')
+  .content
+  .querySelector('a');
+
+const drawPicture = function () {
```

Comment:
Добавь аргумент (функция будет принимать данные одной фотографии)
```suggestion
const drawPicture = function (picture) {
```